app/assets/src/components/views/samples/ToolbarIcon.jsx

```diff
@@ -0,0 +1,67 @@
+// ToolbarIcons are used at the top of the Samples View table.
+// ToolbarIcons are opinionated about their sizing (styling is hard-coded and tweaked for certain icons)
+// If you need multiple sizes of ToolbarIcons, please add a "size" prop instead of directly applying
+// styles to ToolbarIcons.
+
+import React from "react";
+import cx from "classnames";
+import PropTypes from "prop-types";
+
+import BasicPopup from "~/components/BasicPopup";
+import DownloadIcon from "~ui/icons/DownloadIcon";
+import HeatmapIcon from "~ui/icons/HeatmapIcon";
+import PhyloTreeIcon from "~ui/icons/PhyloTreeIcon";
+import SaveIcon from "~ui/icons/SaveIcon";
+
+import cs from "./toolbar_icon.scss";
+
+const ICON_COMPONENTS = {
```

Comment:
Why cant we receive an icon directly? Is it because they have different shapes and need adjustments? 
If it is the second, we should probably fix the icons.

Reply:
I did it mainly to keep it simple when it's being used.

```
<ToolbarIcon icon="save" />
```


instead of 
```
const saveIcon = <SaveIcon />
<ToolbarIcon icon={saveIcon} />
```

The customizability of letting you pass in whatever icon you want didn't seem that important, since we're trying to standardize styles across IDseq, which means restricting the number of options. We should encourage developers to use the pre-styled version.

Reply:
I actually disagree a bit. But it depends on how its going to be used. If this is the only way to use these icons, than it is good design, but if we start adding other "wrappers" to icons depending on location, it becomes hard to maintain... i.e. everytime you add a new icon you need to change multiple places.

Reply:
That makes sense about having multiple wrappers, and referencing the icon component in each of them. I will make the change.